<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyTestProject</class>
 <widget class="QMainWindow" name="MyTestProject">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1599</width>
    <height>966</height>
   </rect>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <widget class="QWidget" name="leftPanel">
       <layout class="QVBoxLayout" name="leftLayout">
        <item>
         <widget class="QTableWidget" name="twTemplates">
          <column>
           <property name="text">
            <string>模板名称</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>宽度</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>高度</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="templateBtnLayout">
          <item>
           <widget class="QPushButton" name="btnAddTemplate">
            <property name="text">
             <string>新增模板</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnRemoveTemplate">
            <property name="text">
             <string>删除模板</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnEditTemplate">
            <property name="text">
             <string>修改模板</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QGroupBox" name="grpTemplate">
          <property name="title">
           <string>模板编辑</string>
          </property>
          <layout class="QFormLayout" name="formTemp">
           <item row="0" column="0">
            <widget class="QLabel" name="lblTemplateName">
             <property name="text">
              <string>名称</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="leTemplateName"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="lblTemplateWidth">
             <property name="text">
              <string>宽度</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QSpinBox" name="sbTemplateWidth">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>10000</number>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="lblTemplateHeight">
             <property name="text">
              <string>高度</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QSpinBox" name="sbTemplateHeight">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>10000</number>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <layout class="QHBoxLayout" name="saveTempLayout">
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Orientation::Horizontal</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Policy::Expanding</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="btnSaveTemplate">
               <property name="text">
                <string>保存模板</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="rightPanel">
       <layout class="QVBoxLayout" name="rightLayout">
        <item>
         <widget class="QGraphicsView" name="graphicsView"/>
        </item>
        <item>
         <widget class="QGroupBox" name="grpPort">
          <property name="title">
           <string>端口组管理</string>
          </property>
          <layout class="QFormLayout" name="formPort">
           <item row="0" column="0">
            <widget class="QLabel" name="lblPortGroupName">
             <property name="text">
              <string>端口组名称</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="lePortName"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="lblPortX">
             <property name="text">
              <string>X</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QSpinBox" name="sbPortX">
             <property name="maximum">
              <number>10000</number>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="lblPortY">
             <property name="text">
              <string>Y</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QSpinBox" name="sbPortY">
             <property name="maximum">
              <number>10000</number>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="lblPortWidth">
             <property name="text">
              <string>宽度</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QSpinBox" name="sbPortWidth">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>10000</number>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="lblPortHeight">
             <property name="text">
              <string>高度</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QSpinBox" name="sbPortHeight">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>10000</number>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <layout class="QHBoxLayout" name="portBtnLayout">
             <item>
              <widget class="QPushButton" name="btnAddPortGroup">
               <property name="text">
                <string>添加端口组</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnRemovePortGroup">
               <property name="text">
                <string>删除选中端口组</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer2">
               <property name="orientation">
                <enum>Qt::Orientation::Horizontal</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Policy::Expanding</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
           <widget class="QTableWidget" name="twPortGroups">
             <column>
                 <property name="text">
                     <string>端口组名称</string>
                 </property>
             </column>
             <column>
                 <property name="text">
                     <string>X</string>
                 </property>
             </column>
             <column>
                 <property name="text">
                     <string>Y</string>
                 </property>
             </column>
             <column>
                 <property name="text">
                     <string>宽度</string>
                 </property>
             </column>
             <column>
                 <property name="text">
                     <string>高度</string>
                 </property>
             </column>
           </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1599</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDesign">
    <property name="title">
     <string>Design</string>
    </property>
   </widget>
   <addaction name="menuDesign"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <!-- <resources/>-->
 <!-- <connections/>-->
</ui>
